src/datachain/lib/dc.py

```diff
@@ -1943,14 +1943,16 @@ def from_csv(
         session: Optional[Session] = None,
         settings: Optional[dict] = None,
         column_types: Optional[dict[str, "Union[str, ArrowDataType]"]] = None,
+        parse_options: Optional[dict[str, "Union[str, bool]"]] = None,
```

Comment:
According to https://arrow.apache.org/docs/python/generated/pyarrow.csv.ParseOptions.html for `invalid_row_handler` key in parse_options value type will be `callable`.